rebar.config

```diff
@@ -4,9 +4,10 @@
 {cover_enabled, true}.
 {cover_export_enabled, true}.
 
-{erl_opts,  [{parse_transform, lager_transform}] }.
+{erl_opts,  [{parse_transform, lager_transform}, {parse_transform, stacktrace_transform}] }.
 
 {deps_dir, "deps"}.
 {deps, [
         {lager, ".*", {git, "https://github.com/erlang-lager/lager.git", {tag, "3.2.4"}}}
+        {stacktrace_compat, ".*", {git, "https://github.com/g-andrade/stacktrace_compat.git", {tag, "1.0.0"}}}
```

Comment:
I think you missed a coma before this line.

Reply:
Indeed, I type so fast that I compile before saving and removing extra chars, like that missing comma :D